tsconfig.options.json

```diff
@@ -3,6 +3,7 @@
   "extends": "tsconfig-moon/tsconfig.json",
   "compilerOptions": {
     "allowJs": true,
+    "rootDir": ".",
```

Comment:
without this the nestjs-app tsconfig won't be able to see the paths outside of the nestjs-app directory.

Doing this allows the app to build.

But not run. because this also affects the structure of the files created in the `nestjs-app/build` directory.

Not sure if I have to solve this with nestjs configuration or if it can be solved with moon or typescript config.